options/internal/generic/allocator.cpp

```diff
@@ -33,3 +36,130 @@ void VirtualAllocator::unmap(uintptr_t address, size_t length) {
 	__ensure(!mlibc::sys_anon_free((void *)address, length));
 }
 
+#else
+
+struct AllocatorMeta {
+	uint64_t allocatedSize;
+	uint64_t pagesSize;
+	frg::array<uint64_t, 4> magic;
+};
+
+constexpr frg::array<uint64_t, 4> __mlibc_allocatorMagic {
```

Comment:
This doesn't really need to have a double underscore, does it?

Reply:
Oh yeah this can just go into an anonymous namespace.